ERROR: Can't guess frontend for input file `-ql' (missing -f option)!

## Steps to reproduce the issue

```bash
make test &> test.log
```

## Expected behavior

No errors

## Actual behavior

A bunch of log lines which ends with the following error (see also the full log [here](https://github.com/YosysHQ/yosys/files/5246452/test.log))

```
cd tests/aiger && bash run-test.sh ""
Checking and_.aag.
Checking buffer.aag.
Checking cnt1.aag.
Warning: The new network has no primary inputs. It is recommended
to add a dummy PI to make sure all commands work correctly.
Checking cnt1e.aag.
Checking empty.aag.
Warning: The new network has no primary inputs. It is recommended
to add a dummy PI to make sure all commands work correctly.
Warning: The current network has no primary outputs. Some commands may not work correctly.
Checking false.aag.
Warning: The new network has no primary inputs. It is recommended
to add a dummy PI to make sure all commands work correctly.
Checking halfadder.aag.
Checking inverter.aag.
Checking notcnt1.aag.
Warning: The new network has no primary inputs. It is recommended
to add a dummy PI to make sure all commands work correctly.
Checking notcnt1e.aag.
Checking or_.aag.
Checking symbols.aag.
Checking toggle-re.aag.
Checking toggle.aag.
Warning: The new network has no primary inputs. It is recommended
to add a dummy PI to make sure all commands work correctly.
Checking true.aag.
Warning: The new network has no primary inputs. It is recommended
to add a dummy PI to make sure all commands work correctly.
Checking and_.aig.
Checking buffer.aig.
Checking cnt1.aig.
Warning: The new network has no primary inputs. It is recommended
to add a dummy PI to make sure all commands work correctly.
Checking cnt1e.aig.
Checking empty.aig.
Warning: The new network has no primary inputs. It is recommended
to add a dummy PI to make sure all commands work correctly.
Warning: The current network has no primary outputs. Some commands may not work correctly.
Checking false.aig.
Warning: The new network has no primary inputs. It is recommended
to add a dummy PI to make sure all commands work correctly.
Checking halfadder.aig.
Checking inverter.aig.
Checking notcnt1.aig.
Warning: The new network has no primary inputs. It is recommended
to add a dummy PI to make sure all commands work correctly.
Checking notcnt1e.aig.
Checking or_.aig.
Checking symbols.aig.
Checking toggle-re.aig.
Checking toggle.aig.
Warning: The new network has no primary inputs. It is recommended
to add a dummy PI to make sure all commands work correctly.
Checking true.aig.
Warning: The new network has no primary inputs. It is recommended
to add a dummy PI to make sure all commands work correctly.
Running neg.ys.

 /----------------------------------------------------------------------------\
 |                                                                            |
 |  yosys -- Yosys Open SYnthesis Suite                                       |
 |                                                                            |
 |  Copyright (C) 2012 - 2020  Claire Wolf <claire@symbioticeda.com>          |
 |                                                                            |
 |  Permission to use, copy, modify, and/or distribute this software for any  |
 |  purpose with or without fee is hereby granted, provided that the above    |
 |  copyright notice and this permission notice appear in all copies.         |
 |                                                                            |
 |  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES  |
 |  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF          |
 |  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR   |
 |  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES    |
 |  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN     |
 |  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF   |
 |  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.            |
 |                                                                            |
 \----------------------------------------------------------------------------/

 Yosys 0.9+3575 (git sha1 7affef7c, gcc 7.5.0_1 -fPIC -Os)


-- Executing script file `neg.ys' --

1. Executing Verilog-2005 frontend: <<EOT
Parsing Verilog input from `<<EOT' to AST representation.
Generating RTLIL representation for module `\top'.
Successfully finished Verilog frontend.

2. Shell command: rm -rf neg.out

3. Shell command: mkdir neg.out

4. Executing SIMPLEMAP pass (map simple cells to gate primitives).
Mapping top.$and$<<EOT:2$1 ($and).

5. Executing AIGER backend.

6. Executing AIGER frontend.
<suppressed ~385 debug messages>
Removed 0 unused cells and 192 unused wires.

7. Shell command: rm -rf neg.out

8. Shell command: mkdir neg.out

9. Executing SIMPLEMAP pass (map simple cells to gate primitives).
Mapping top.$and$<<EOT:2$1 ($and).

10. Executing XAIGER backend.
<suppressed ~5 debug messages>

11. Executing AIGER frontend.
<suppressed ~385 debug messages>
Removed 0 unused cells and 192 unused wires.
ERROR: Can't guess frontend for input file `-ql' (missing -f option)!
make: *** [test] Error 1
```

--- a/tests/aiger/run-test.sh
+++ b/tests/aiger/run-test.sh
@@ -55,5 +55,5 @@ done
 
 for y in *.ys; do
     echo "Running $y."
-    ../../yosys $y -ql ${y%.*}.log
+    ../../yosys -ql ${y%.*}.log $y
 done

